### DISCUSSION TITLE
How to fix browser error "Invalid UI version in the JSON-RPC response" when running MinIO compiled by source code on Windows

### DISCUSSION THREAD

**Discussion by Jiang765** [minio/minio#12517] [Q&A] [answered]
Hi! I downloaded one of the release versions and compiled the source code. Then I opened the browser using 127.0.0.1:9000. There is an error saying "Invalid UI version in the JSON-RPC response".

I am not sure how to fix the problem. Could anyone give me some advice? Thank you very much!

**Comment by harshavardhana** [comment, marked as answer]
You need to compile it with some proper release values, this has never been supported.

Compile it like `make install` and it will work fine.
